### PR TITLE
Completed prep-css-flexbox exercise.

### PR DESCRIPTION
**Pull request by joey-gaffney**
Screen Shot 2022-06-09 at 8 43 13 PM
Screen Shot 2022-06-09 at 8 43 37 PM
Screen Shot 2022-06-09 at 8 43 58 PM
Screen Shot 2022-06-09 at 8 44 28 PM

### CODE REVIEW

File: prep-css-flexbox/flex-wrap.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+<head>
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck{
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% / 6);
```
Comment (thebearingedge) [inline prep-css-flexbox/flex-wrap.html:12]: This `flex-basis` property should be applied to the `.card-wrapper` element(s) instead of the `#card-deck`

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, great, just one small change 👍

File: prep-css-flexbox/flex-wrap.html (hunk 1 above)
Comment (joey-gaffney) [reply prep-css-flexbox/flex-wrap.html:12]: Okay, fixing now then sending it. Thanks!

Comment (joey-gaffney) [reply prep-css-flexbox/flex-wrap.html]: Screen Shot 2022-06-09 at 9 03 40 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**